zstacklib/zstacklib/gpu/vendors/nvidia.py

```
@@ -594,3 +585,130 @@ def detect_sriov_capability(cls, pci_device_to, gpu_info_map=None):
+        Detect NVIDIA TensorFusion (GPU virtualization) capability.
+
+        Requirements:
+        - NVIDIA driver version >= 570.x
+        - CUDA version >= 12.1
+
+        Returns tuple: (is_supported, capability_info)
+        """
+        addr = pci_device_to.pciDeviceAddress
+
+        # Query GPU details for this specific device
+        r, o, e = bash_roe('nvidia-smi --query-gpu=pci.bus_id,driver_version,cuda_version --format=csv,noheader -i %s' % addr)
```

Ruff (0.15.5)

[warning] Unpacked variable `e` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)